profiles/Weeks-setup.yaml

```diff
@@ -338,25 +464,78 @@ safe_room: 851
 #- skippet
 #- crate
 #- box
+- cambrinth lump
+- package
+- sraeth runestone
+- gold ring
+- kirmhiro draught
+- signet ring
+- witch ball
+- satin pillow
+- license
+- writ
+- permit
+- pass
+- profile
+- seastars
+- pendant
+- card
+- jasper
+- heliodor
+- tasialm
+- seastar
+- papyrus roll
+- gold nugget
+- platinum nugget
+- lumium nugget
+- ninium nugget
+- muracite nugget
+- darkstone nugget
+- damite nugget
+- kertig nugget
+- haralun nugget
+- glaes nugget
+- animite nugget
+
 
 listen: true
 
-theurgy_supply_container: burlap bag
+theurgy_supply_container: duffel bag
 flint_lighter: dao
 water_holder: chalice
 
+stealing_bag: pack
+
+held_cambrinth: false
```

Comment:
This setting can be removed. We now assume cambrinth is worn and check your Arcana skill (unless `stored_cambrinth` is true).